src/main/java/service/imp/ShopServiceImpl.java

```diff
@@ -0,0 +1,20 @@
+package service.imp;
```

Comment:
The package declaration should follow the project structure and common practices. Since the `ShopServiceImpl` class is an implementation of a service, it should be placed in the `service.impl` package, not `service.imp`. Please correct the package name.